kaniko.go

```diff
@@ -264,3 +283,60 @@ func getDigest(digestFile string) string {
 func trace(cmd *exec.Cmd) {
 	fmt.Fprintf(os.Stdout, "+ %s\n", strings.Join(cmd.Args, " "))
 }
+
+func getPluginEnvVars(cmdArgs []string) []string {
+	envVars := os.Environ()
+
+	// Iterate through environment variables
+	for _, envVar := range envVars {
+		// Check if the variable starts with PLUGIN_
```

Comment:
Nit: update comment
// Check if the variable starts with PLUGIN_ and is also present in the allow list

Reply:
done